src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -37,17 +43,45 @@ class LicenceConditionService(
     val entityStandardPssConditions = request.standardPssConditions.transformToEntityStandard(licenceEntity, "PSS")
 
     val username = SecurityContextHolder.getContext().authentication.name
+    val createdBy = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
+      ?: throw RuntimeException("Staff with username $username not found")
 
     val updatedLicence = licenceEntity.copy(
       standardConditions = entityStandardLicenceConditions + entityStandardPssConditions,
       dateLastUpdated = LocalDateTime.now(),
       updatedByUsername = username,
     )
+
+    /*
+    When the function is called, the standard conditions are updated in line with the current policy version but
+    the licence will retain the version of the policy it was created with because any additional conditions will be on a previous
+    version and not updated. These additional conditions can only be updated via the vary journey.
```

Comment:
This is only true in the context of how the fronted currently interacts with the backend through it's endpoint. Not sure this kind of comment belongs in the service layer of the API.  

(Not a massive fan of lots of comments in the code generally, they get out of date and are a pain to maintain - it would be nice to have some documentation about how versioning works but maybe outside the codebase)

Reply:
Whoops, another one I forgot to take that out - removed it from the code.